beta-src/tsconfig.json

```diff
@@ -1,11 +1,7 @@
 {
```

Comment:
These seem like unnecessary changes to the file, how did they come about? This should probably be reverted.